lib/rspec/matchers/built_in/raise_error.rb

```diff
@@ -160,6 +160,7 @@ def warn_about_bare_error
                         "will match when Ruby raises a `NoMethodError`, `NameError` or " \
                         "`ArgumentError`, potentially allowing the expectation to pass " \
                         "without even executing the method you are intending to call. " \
+                        "Actual error raised was #{description_of(@actual_error)}. "\
```

Comment:
is there a test already covering when `@actual_error` is `nil`? (If the spec passes?) Intuitively seems like the output would be wrong with current code.